docs/content_management/block_reference.md

```diff
@@ -141,6 +142,19 @@ On the **Basic** tab, set values in the following fields:
 After submitting the settings, all images in the folder will appear in the Gallery block.
 Note that selecting a folder containing Content items other than images results in displaying only a link to the folder they are stored in.
 
+## Ibexa Connect block
+
+On the **Basic** tab, set values in the following fields:
```

Comment:
```suggestion
In the **Basic** tab, set values in the following fields:
```

Reply:
Let's keep "on" for the page consistency.